dispatcher/attributes.go

```diff
@@ -41,7 +39,6 @@ func (dS *DispatcherService) AttributeSv1GetAttributeForEvent(args *CGREvWithApi
 		Tenant:  args.Tenant,
 		ID:      utils.UUIDSha1Prefix(),
 		Context: utils.StringPointer(utils.MetaAuth),
-		Time:    utils.TimePointer(time.Now()),

```

Comment:
could it be that you have removed by mistake?
Normally time should be taken from the event side coming in to Dispatcher